apps/vr-tests-react-components/package.json

```diff
@@ -46,6 +46,7 @@
     "@fluentui/react-progress": "^9.1.19",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 49 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28323&commitId=0f8ec3f916e8373eee2e1cac5c3c3e95795d02d4&env=prod)

✅ There was 49 screenshots added, 0 screenshots removed, 2040 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 49 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox Converged.Appearance- filled-darker.default.chromium.png|0| Added|
|SearchBox Converged.Appearance- filled-darker.focused.chromium.png|0| Added|
|SearchBox Converged.Appearance- filled-darker.hover.chromium.png|0| Added|
|SearchBox Converged.Appearance- filled-lighter.default.chromium.png|0| Added|
|SearchBox Converged.Appearance- filled-lighter.focused.chromium.png|0| Added|
|SearchBox Converged.Appearance- filled-lighter.hover.chromium.png|0| Added|
|SearchBox Converged.Appearance- outline (default).default.chromium.png|0| Added|
|SearchBox Converged.Appearance- outline (default).focused.chromium.png|0| Added|
|SearchBox Converged.Appearance- outline (default).hover.chromium.png|0| Added|
|SearchBox Converged.Appearance- underline.default.chromium.png|0| Added|
|SearchBox Converged.Appearance- underline.focused.chromium.png|0| Added|
|SearchBox Converged.Appearance- underline.hover.chromium.png|0| Added|
|SearchBox Converged.Clears value.default.chromium.png|0| Added|
|SearchBox Converged.Clears value.dismiss clicked.chromium.png|0| Added|
|SearchBox Converged.Clears value.dismiss focused.chromium.png|0| Added|
|SearchBox Converged.Clears value.input focused.chromium.png|0| Added|
|SearchBox Converged.Disabled, appearance- filled-darker.default.chromium.png|0| Added|
|SearchBox Converged.Disabled, appearance- filled-darker.focused.chromium.png|0| Added|
|SearchBox Converged.Disabled, appearance- filled-darker.hover.chromium.png|0| Added|
|SearchBox Converged.Disabled, appearance- filled-lighter.default.chromium.png|0| Added|
|SearchBox Converged.Disabled, appearance- filled-lighter.focused.chromium.png|0| Added|
|SearchBox Converged.Disabled, appearance- filled-lighter.hover.chromium.png|0| Added|
|SearchBox Converged.Disabled, appearance- outline.default.chromium.png|0| Added|
|SearchBox Converged.Disabled, appearance- outline.focused.chromium.png|0| Added|
|SearchBox Converged.Disabled, appearance- outline.hover.chromium.png|0| Added|
|SearchBox Converged.Disabled, appearance- underline.default.chromium.png|0| Added|
|SearchBox Converged.Disabled, appearance- underline.focused.chromium.png|0| Added|
|SearchBox Converged.Disabled, appearance- underline.hover.chromium.png|0| Added|
|SearchBox Converged.Size- large.default.chromium.png|0| Added|
|SearchBox Converged.Size- large.focused.chromium.png|0| Added|
|SearchBox Converged.Size- large.hover.chromium.png|0| Added|
|SearchBox Converged.Size- small.default.chromium.png|0| Added|
|SearchBox Converged.Size- small.focused.chromium.png|0| Added|
|SearchBox Converged.Size- small.hover.chromium.png|0| Added|
|SearchBox Converged.WIth contentAfter - RTL.default.chromium.png|0| Added|
|SearchBox Converged.WIth contentAfter - RTL.focused.chromium.png|0| Added|
|SearchBox Converged.WIth contentAfter - RTL.hover.chromium.png|0| Added|
|SearchBox Converged.WIth contentAfter.default.chromium.png|0| Added|
|SearchBox Converged.WIth contentAfter.focused.chromium.png|0| Added|
|SearchBox Converged.WIth contentAfter.hover.chromium.png|0| Added|
|SearchBox Converged.With custom dismiss - RTL.default.chromium.png|0| Added|
|SearchBox Converged.With custom dismiss - RTL.focused.chromium.png|0| Added|
|SearchBox Converged.With custom dismiss - RTL.hover.chromium.png|0| Added|
|SearchBox Converged.With custom dismiss.default.chromium.png|0| Added|
|SearchBox Converged.With custom dismiss.focused.chromium.png|0| Added|
|SearchBox Converged.With custom dismiss.hover.chromium.png|0| Added|
|SearchBox Converged.With value.default.chromium.png|0| Added|
|SearchBox Converged.With value.focused.chromium.png|0| Added|
|SearchBox Converged.With value.hover.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>